tests/specs/account.spec.js

```diff
@@ -25,14 +26,50 @@ describe("Account", () => {
         await elemSelectControlInput.addValue(currency);
         const elemFirstOption = await $(".select-options-list .option:first-child");
         await elemFirstOption.click();
-        const elemContinueButton = await $("#modal-continue-button");
-        await elemContinueButton.click();
-        await mockDeviceEvent({ type: "opened" });
-        const elemImportAddButton = await $("#add-accounts-import-add-button");
-        await elemImportAddButton.waitForDisplayed();
-        await elemImportAddButton.waitForEnabled();
-        await elemImportAddButton.click();
-        await modalPage.close();
+        if (currency === "chainlink") {
+          // Add token when parent missing
+          const elemAddParentButton = await $("#modal-token-continue-button");
+          await elemAddParentButton.waitForDisplayed();
+          expect(await app.client.screenshot()).toMatchImageSnapshot({
+            customSnapshotIdentifier: "addTokenWithoutParent",
```

Comment:
we should have only one screenshot test per tests and the screenshot test should be at the end of the test so that if it fails it does not break what is following